app/src/main/java/com/sdp/movemeet/ProfileActivity.java

```diff
@@ -131,8 +131,12 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
             case R.id.nav_chat:
                 Navigation.goToChat(this.navigationView);
                 break;
+            case R.id.nav_list_activities:
+                Navigation.goToListOfActivities(this.navigationView);
```

Comment:
+ `finish()` (I hope these are actually required, or I'm just marking them for nothing)

Reply:
Done in last commit